GUIDELINES.md

```diff
@@ -123,7 +124,7 @@ All Blog posts are located in the [blog/\_posts](blog/_posts/) directory, and al
 
 ### Pages
 
-_[Pages](https://jekyllrb.com/docs/pages/)_ are located at the [pages](/pages/) directory, to create one follow these steps:
+The _[Pages](https://jekyllrb.com/docs/pages/)_ are located at the [pages](/pages/) directory, to create one follow these steps:
```

Comment:
Added "The" to make it stop complaining about "repeated word". Same below for "Labs".